solnsumei/hello-books
# #149783603 Google Signup

#### What does this PR do?
The PR adds google sign up functionality
#### Description of Task to be completed?
- install react-google-login package
- implement google login on client-side
- add method to check google user on the server side
#### How should this be manually tested?
- When user navigates to the login or signup pages
- Then they should see a button to sign up or sign in using google
#### What are the relevant pivotal tracker stories?
- #149783603 User should be able to signup with Google+

[Delivers #149783603]
